app/timetables/models.py

```diff
@@ -39,3 +39,20 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
+    """
+    Model representing Dish.
+    A dish represents the actual food served as a given
+    course under an option of a meal on a cycle day in a timetable.
+    E.g, Coconut rice garnished with fish stew and chicken or just
+    Ice-cream.
+    """
+
+    name = models.CharField(max_length=255, unique=True)
+
```

Comment:
@andela-joyebanji Well done for your effort. Though this a new development, it would be nice if work on this issue as a whole is completed ASAP.